frontend/components/command.tsx

```diff
@@ -0,0 +1,16 @@
+export const Command = ({ text }: { text: string }) => (
+  <div className="flex flex-col sm:flex-row items-baseline justify-between">
+    <div className="mb-4 flex overflow-auto text-[1rem] sm:text-[1em]">
```

Comment:
Did you try with `overflow-scroll md:overflow-auto`? I think I prefer the font to have the same size as the other text :)

Reply:
I did, the issue here is that while I can get the scroll bars to appear with `overflow-scroll`, the content inside is not scrollable because it is overflowing it's bounding box. I was trying to figure that out, but wanted to avoid changing too much code. You can see the issue here:

![image](https://user-images.githubusercontent.com/29112081/170622882-31057289-62f0-47ad-9138-12ac98a9ce58.png)

The grid extends beyond the `Box`. So even if we get those scroll bars, the content won't actually be scrollable. With the `overflow-auto` we will get scroll bars once the bounding box of the content requires the overflow to scroll. 
My hunch is that the text on the left is as narrow as it will go because of word breakpoints, it has padding. Those together set the `1fr` and the `2fr` width of the right part of the grid is based on that. That is wider than the `Box` and causes the content to overflow.

Reply:
Let me know what you end up thinking with this :) Happy to make any changes you need.